addons/binding/org.openhab.binding.yamahareceiver/ESH-INF/thing/thing-types.xml

```diff
@@ -4,21 +4,19 @@
         xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0"
         xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
 
-    <!-- Sample Thing Type -->
-    <thing-type id="yamahaAV">
+    <bridge-type id="yamahaAV">
         <label>Yamaha Receiver</label>
         <description>Yamaha Receiver of product line CX-A5000, RX-A30xx, RX-A20xx, RX-A10xx, RX-Vxxx, RX-Z7, DSP-Z7, RX-S600, HTR-xxxx</description>
 
         <channels>
             <channel id="power" typeId="power"/>
-            <channel id="netradiotune" typeId="netradiotune"/>
-            <channel id="input" typeId="input"/>
-            <channel id="surroundProgram" typeId="surroundProgram"/>
-            <channel id="volume" typeId="volume"/>
-            <channel id="volumeDB" typeId="volumeDB"/>
-            <channel id="mute" typeId="mute"/>
         </channels>
         
+        <properties>
+        <property name="version">0.0</property>
```

Comment:
indentation wrong.
The values do not seem to be relevant - you should rather remove those properties (you can still set them through code)